execution_engine/src/core/runtime/mint_internal.rs

```diff
@@ -67,6 +67,29 @@ where
     fn get_main_purse(&self) -> URef {
         self.context.account().main_purse()
     }
+
+    fn is_account_administrator(&self, account_hash: &AccountHash) -> Option<bool> {
+        self.config.is_account_administrator(account_hash)
+    }
+
+    fn can_perform_unrestricted_transfer(&self) -> bool {
```

Comment:
Maybe for consistency:

```suggestion
    fn allow_unrestricted_transfers(&self) -> bool {
```